include/fx/gltf.h

```diff
@@ -1697,6 +1708,20 @@ namespace gltf
                     std::memcpy(&buffer.data[0], &binary[HeaderEndOffset], buffer.byteLength);
                 }
             }
+            if (dataContext.extraChunksLength != 0)

```

Comment:
In general, leave a blank line after each closing }.  This goes for several other places in the code as it seems clang-format doesn't have an option to enforce this...